x11-plugins/purple-libnotify-plus/purple-libnotify-plus-2.99.1.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
```

Comment:
Requires a revbump and

Reply:
@juippis Thanks for the review, I'm working with @SoapGentoo on gcc-10 fixes, I don't think these changes would cause any additional regressions to need a revbump, so I'd try to avoid revbumping unnecesarily. What do you think?

Reply:
Sure, the proxy-maint project don't have time to go through that thorough testing so we always revbump, but if @SoapGentoo takes the responsibility I don't mind.

But do see the comments about incompatible eclasses with EAPI-7.